src/sql/azdata.proposed.d.ts

```diff
@@ -147,22 +147,20 @@ declare module 'azdata' {
 
 	export interface RadioCard {
 		id: string;
-		label: string;
-		descriptions?: RadioCardDescription[];
+		descriptions: RadioCardDescription[];
 		icon?: string | vscode.Uri | { light: string | vscode.Uri; dark: string | vscode.Uri };
 	}
 
 	export interface RadioCardDescription {
-		ariaLabel: string;
-		labelHeader: string;
-		contents: RadioCardLabelValuePair[];
-		valueHeader?: string;
+		textValue: string;
+		linkDisplayValue?: string;
+		displayLinkCodicon?: boolean;
+		textStyles?: CssStyles;
+		linkStyles?: CssStyles;
+		linkCodiconStyles?: CssStyles;
 	}
 
-	export interface RadioCardLabelValuePair {
-		label: string;
-		value?: string;
-	}
+	export type CssStyles = { [key: string]: string };
```

Comment:
Probably should make this a stricter type, since not all strings are valid as css.

Reply:
Uuuuh what would you recommend? Maybe there is some library out there that provides specific CSS typings?

Reply:
See if you like the strategy now :)

Reply:
I'm actually removing what I had added.